deps/exokit-bindings/webglcontext/include/webgl.h

```diff
@@ -85,6 +85,277 @@ enum GlKey {
   GL_KEY_PLANE,
 };
 
+#define GL_READ_BUFFER                                  0x0C02
```

Comment:
How many of these are actually needed? I suspect that the number is low since the platform is providing these. In that case, I would defer to the platform versions since those are the exact values we pass back to the platform when we do make WebGL calls.